src/locations.jl

```diff
@@ -109,10 +109,8 @@ Determines the location to save a datadep with the given name to.
 """
 function determine_save_path(name, rel=nothing)::String
     cands = preferred_paths(rel; use_package_dir=false) #TODO Consider removing `rel` argument, it is not used
-    path_ind = findfirst(cands) do path
-        0 == first(uv_access(path, W_OK))
-    end
-    if path_ind==0
+    path_ind = findfirst(can_read_file, cands)
```

Comment:
This is not the same
`can_read_file` checks for `R_OK`,
but we need to check `W_OK`, 
like in `uv_access(path, W_OK)`

Reply:
Got it - reverted